modules/social_features/social_group/social_group.api.php

```diff
@@ -379,6 +379,36 @@ function hook_social_group_cross_posting_alter(array &$content_types) {
   $content_types[] = 'topic';
 }
 
+/**
+ * Provide entity types in which entities can be linked to the group.
+ *
+ * @return array
+ *   An associative array of statuses that shows if additional fields per each
+ *   group type should be created, keyed by entity type identifier.
+ *
+ * @see group_entity_base_field_info()
+ */
+function hook_group_types(): array {
+  return [
+    'node' => TRUE,
+    'user' => FALSE,
+  ];
+}
+
+/**
+ * Alter entity types in which entities can be linked to the group.
+ *
+ * @param array $types
+ *   An associative array of entity types returned by hook_group_types().
+ *
+ * @see group_entity_base_field_info()
+ */
+function hook_group_types_alter(array &$types): void {
+  if (isset($types['user']) && !$types['user']) {
+    $types['user'] = TRUE;
+  }
+}
+
```

Comment:
I think, considering it's part of `open_social` it's best to write these hooks from a `hook_social_` naming, so it's clear people don't have to look in group but in social_group, although there is already a hook with that name.

But I'm also wondering what the value is for both of these hooks, the fact they live in a entity base field info (and alter) themselves, doesn't that mean others should be able to similarly use an field info alter, with a correct weight, to change base fields themselves?

Reply:
I suppose it was just moved from the patch that we used before: https://www.drupal.org/files/issues/2022-07-07/group-computed-fields-2718195-65.patch

and for the current moment if we can I would postpone renaming of those hooks, because it uses in the other extensions to enable/disable group feature and it will cause a new bunch of changes in the related PR